core/server/worker/src/main/java/alluxio/worker/block/DefaultBlockWorker.java

```diff
@@ -313,7 +314,7 @@ public void commitBlock(long sessionId, long blockId, boolean pinOnCreate)
     }
     BlockMasterClient blockMasterClient = mBlockMasterClientPool.acquire();
     try {
-      BlockMeta meta = mLocalBlockStore.getVolatileBlockMeta(blockId);
+      BlockMeta meta = mLocalBlockStore.getVolatileBlockMeta(blockId).get();
```

Comment:
isPresent?

Reply:
sometimes we directly call `get()` on optional without checking because at the code path the optional is expected to have value. Basically if we check for `isPresent` and would throw an exception if the value is not present then the check is not needed, because `get()` will throw exception anyways. The code is less verbose this way. It's one of the warnings that I tend to ignore.